blog/content/post/2021-12-13-secure-jtag-on-imx-soc.md

```diff
@@ -0,0 +1,280 @@
+---
+title: 'Increasing the security of iMX platforms - JTAG fusing'
+abstract: 'JTAG helps a lot of engineers during product development. It also may
+          be helpful for hackers. I tell you why and how to increase JTAG
+          security in your product'
+cover: /img/secure_jtag_cover.png
+author: michal.kotyla
+layout: post
+published: true
+date: 2021-12-13
+archives: "2021"
+
+tags:
+  - security
+  - jtag
+  - fusebits
+  - fusing
+  - nxp
+  - i.mx6
+categories:
+  - Security
+  - Manufacturing
+
+---
+## Introduction
+
+JTAG port is an important feature that helps a lot of engineers during product
+development. It also may be helpful for hackers while performing a
+reverse-engineering process - e.g. they can dump the memory of your firmware and
+get access to strictly confidential information. In this article, we show how to
+lock JTAG access for users who don't have a special key. We use for that Secure
+JTAG feature which is implemented on i.MX SoC's. In our lab, we tested this
+solution on i.MX6 but this same feature is available on a lot more NXP
+platforms. Unfortunately they have missing public documentation.
+
+## Requirements
+
+- **Development board**: Any devkit with i.MX6.
```

Comment:
Does it mean hardware design not matter at all? I assume there is need at least for JTAG pins exposure?

Reply:
Of course, JTAG pins are needed. I will add that